app/models/assets/import.rb

```diff
@@ -309,19 +309,11 @@ def import_remote_asset(remote_asset, barcode, import_step)
       Asset
         .create!(barcode: barcode, uuid: remote_asset.uuid, facts: [])
         .tap do |asset|
-          FactChanges
-            .new
-            .tap do |updates|
-              updates.replace_remote(asset, 'a', sequencescape_type_for_asset(remote_asset))
-              updates.replace_remote(asset, 'remoteAsset', remote_asset.uuid)
-            end
-            .apply(import_step)
-
```

Comment:
These facts got immediately replaced by the following steps, so were largely redundant.